addons/website_theme_install/static/src/less/website_theme_install.less

```diff
@@ -1,42 +1,164 @@
+@o-theme-kanban-gray: #fcfcfc;
+
 .o_kanban_view.o_theme_kanban {
+    background-color: @o-theme-kanban-gray;
+
+    /// Un-grouped Layout (default)
     &.o_kanban_ungrouped {
         .o-justify-content(center);
 
         .o_kanban_record {
-            .o-flex(0, 0, auto);
-            width: 100%;
-            @media (min-width: @screen-sm-min) {
-                width: 45%;
-            }
+            .o-flex(0, 1, auto);
+            width: 250px;
         }
     }
 
     .o_kanban_record {
-        padding: 0;
+        margin: 20px 15px;
+        box-shadow: none;
+        background: none;
+        border: none;
+
+        .o_browser_bar:before {
+            content: "•••";
+            display: block;
+            padding-left: 10px;
+            color: darken(@o-theme-kanban-gray, 10%);
+            font: 12px/20px Courier, monospace;
+            letter-spacing: -1px;
+        }
+
+        &.o_theme_installed .o_theme_preview_top {
+            border-color: @odoo-brand-optional;
+
+            &::after {
+                .o-position-absolute(10px, 10px);
+                .square(20px);
+                display: block;
+                line-height: 20px;
+                text-align: center;
+                background-color: @odoo-brand-optional;
+                border-radius: 100%;
+                font-family: 'FontAwesome';
+                color: white;
+                content: "\f00c";
```

Comment:
Could you maybe put a `<span/>` element during rendering (with right `fa fa-...`) instead of using LESS style for that ? :)